### ISSUE TITLE
Change custom Event Grid eventType property from an int to a string

### ISSUE THREAD

**Issue by cgillum** [Azure/azure-functions-durable-extension#285]
The data/eventType field in the custom Event Grid notification is currently an int value, representing the orchestration runtime status. This is confusing because:

1. There is already an **eventType** field in the parent JSON object which has a different value.
2. It's not really an "event type" but rather the runtime status of the orchestration
3. Integers are hard to understand when used for status

Here is an example of the current paylod:
```json
[
  {
    "id": "aeafd489-ee98-4b2e-b77f-2aac2594b4b2",
    "topic": null,
    "subject": "durable/orchestrator/Running",
    "data": {
      "hubName": "DFHub",
      "functionName": "Counter",
      "instanceId": "13b602a9d2914d7c981016acbee0b941",
      "reason": "",
      "eventType": 0
    },
    "eventType": "orchestratorEvent",
    "eventTime": "2018-04-29T04:41:43.5299974Z",
    "metadataVersion": null,
    "dataVersion": "1.0"
  }
]
```

My proposal is to change **data/eventType** to **data/runtimeStatus** and make it a `string` instead of an `int`. Here is the same example, but translated:

```json
[
  {
    "id": "aeafd489-ee98-4b2e-b77f-2aac2594b4b2",
    "topic": null,
    "subject": "durable/orchestrator/Running",
    "data": {
      "hubName": "DFHub",
      "functionName": "Counter",
      "instanceId": "13b602a9d2914d7c981016acbee0b941",
      "reason": "",
      "runtimeStatus": "Running"
    },
    "eventType": "orchestratorEvent",
    "eventTime": "2018-04-29T04:41:43.5299974Z",
    "metadataVersion": null,
    "dataVersion": "1.0"
  }
]
```

This will be a breaking change for any consumers of this field. @TsuyoshiUshio, @yuka1984, please let me know if you have any concerns with this change. I can take care of implementing it for GA if you are agreeable to this change.

**Comment by cgillum** [comment]
Another option is to remove this second `eventType` field since it is redundant with the `subject` field.

**Comment by TsuyoshiUshio** [comment]
@cgillum  OK. I'll ask yuka and the guys. I'm ok for both solution, and I can help. Also, we need to care the documentation change. I also send pull request to the documentation as well.

**Comment by TsuyoshiUshio** [comment]
@cgillum They said it’s OK. Let’s do your recommendation. Subject is used for filtering. I’ll do this. However, you remove this from G.A. Milestone. After the GA is fine? Or prefer do it now?

**Comment by cgillum** [comment]
I removed it from GA, but it can still be brought back in. :) It just means there is no rush if you are too busy.

**Comment by TsuyoshiUshio** [comment]
Yeah. It is breaking  change.  It might be good before the GA. I'll try that. with documentation.

**Comment by cgillum** [comment]
The fix has been merged and will go out with the GA release.
